content/docs/state-and-lifecycle.md

```diff
@@ -1,23 +1,24 @@
 ---
 id: state-and-lifecycle
-title: State and Lifecycle
+title: राज्य और जीवनचक्र
```

Comment:
We are not translating `state` you can read more about these at #2 
`राज्य` => `State`